### PR TITLE
add slim support

### PR DESCRIPTION
**Pull request by liubin**
The slim view files is generated by `erb2slim` command that provided by https://github.com/slim-template/html2slim

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,11 @@ And then execute:
 $ rails g rambulance:install
 ```
 
+or specify the template engine by:
```
Comment (yuki24) [inline README.md]: Also mention haml so people will know what templates are available.

File: lib/rambulance/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module Rambulance
-  VERSION = "0.3.0"
+  VERSION = "0.3.1"
```
Comment (yuki24) [inline lib/rambulance/version.rb]: Don't change the version.

**Comment by yuki24** [comment]
Thanks! I was actually thinking about the haml templates but maybe adding support for slim sounds good at this point.

**Comment by liubin** [comment]
I've updated according to the review.

**Comment by yuki24** [comment]
Thanks!

**Comment by yuki24** [comment]
I realized that this PR had 2 commits. Please squash them into one next time you send a pull request.

**Comment by liubin** [comment]
OK.

**Pull request merged**